pkg/kubemark/hollow_kubelet.go

```diff
@@ -141,7 +142,6 @@ func GetHollowKubeletConfig(opt *HollowKubletOptions) (*options.KubeletFlags, *k
 	c.FileCheckFrequency.Duration = 20 * time.Second
 	c.HTTPCheckFrequency.Duration = 20 * time.Second
 	c.NodeStatusUpdateFrequency.Duration = 10 * time.Second
-	c.NodeStatusReportFrequency.Duration = 5 * time.Minute
```

Comment:
@yb01 I don't know the reason behind this line in the poc branch. do you know if we still need this for scale-out when moving to master?

Reply:
the 5 minutes node status report frequency seems align with YingHang has ported from the community. so we should keep it.

Reply:
ok I will file another change to add this back in. thanks.

Reply:
Next time please add the mistakenly removed line back in the same commit. It is better to keep each commit clean by itself. Thanks.